### DISCUSSION TITLE
关于在创建 button 时，on_click 自动执行后面的函数的问题

### DISCUSSION THREAD

**Discussion by umarurize** [EndstoneMC/endstone#83] [Q&A]
![7f4c39023357e6a7d8f6b06e764af66](https://github.com/user-attachments/assets/e272f29c-9745-434f-a807-e250c19b2762)
部分代码如图

问题是我在运行写好的插件，打开表单时，希望在点击后，弹出二级表单（即confirm()函数的内容）。但是，在执行命令时，并没有让我有机会进行点击操作，就直接执行了 on_click 后面的函数。

尝试了很多办法无果，求大佬解答...

**Comment by umarurize** [comment]
@wu-vincent

**Reply by legitbox** [reply]
please use english for your questions!

**Comment by wu-vincent** [comment]
Please use English in all issues, pull requests and discussions for the benefit of everyone. I am going to close it for now and feel free to open another one in English.
